docs/source/markdown/podman-create.1.md

```diff
@@ -248,6 +248,12 @@ If you have four memory nodes on your system (0-3), use `--cpuset-mems=0,1`
 then processes in your container will only use memory from the first
 two memory nodes.
 
+#### **--dep**=**container**
+
+Specify one or more dependency containers.
```

Comment:
How do they specify?  name, ctrid, other?  How are multiples specified?  One per `--dep` option or by comma separated list?  Examples would be very nice.  Same comment through out.

Reply:
same comment for the other commands.